app/konnect/dev-portal/access-and-approval/add-teams.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Add Developer Teams from Identity Providers
+content_type: how-to
+---
+
+In the {{site.konnect_short_name}} Dev Portal, you can import existing developer teams from a third-party identity provider (IdP) and map their permissions to elements in {{site.konnect_short_name}}. This allows you to import developers automatically instead of manually copying over every single team of developers from your IdP to {{site.konnect_short_name}}.
+
+This guide explains how to map the permissions, including scopes and claims, from your group of developers in your IdP to your organization's team in {{site.konnect_short_name}}. Although this guide uses Okta, Azure, and Auth0 as examples, you can use any IdP that conforms to OIDC standards. 
+
+## Prerequisites
+An IdP configured in Konnect (OIDC or DCR) for [application registration](konnect/dev-portal/applications/enable-app-reg/)
+
+## Create developer teams in your IdP
+
+<!-- do these IdPs need the Kong callback URl added to them to be properly configured? Or is this just a one-way pull from the IdP to Konnect, so it doesn't need any information from Konnect?-->
```

Comment:
@c3pko I wasn't sure if the Kong callback URL needed to go anywhere in the IdP, do you happen to know?

Reply:
yes! it should go here https://docs.konghq.com/konnect/dev-portal/applications/enable-app-reg/